.gitignore

```diff
@@ -14,7 +14,7 @@ features/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.feature/src/main/resou
 features/apimgt/org.wso2.carbon.apimgt.rest.api.store.feature/src/main/resources/api#am#store#*.war
 features/apimgt/org.wso2.carbon.apimgt.rest.api.admin.feature/src/main/resources/api#am#admin#*.war
 features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/throttle#data#*.war
-features/apimgt/org.wso2.carbon.apimgt.micro.gateway.default.feature/src/main/resources/micro-gateway#v0.9.war
+features/apimgt/org.wso2.carbon.apimgt.micro.gateway.default.feature/src/main/resources/micro-gateway#v0.10.war
```

Comment:
instead of specifying the version, shall we use * as in other wars?

Reply:
done